internal/backends/collection.go

```diff
@@ -105,17 +105,29 @@ type QueryResult struct {
 // It also can be used to close the returned iterator and free underlying resources,
 // but doing so is not necessary - the handler will do that anyway.
 //
-// If SortPushdown is false, any sort, or limit pushdown won't be performed.
-// If SortPushdown is false, Sort and Limit fields cannot be set.
-// If SortPushdown is true, but Sort is equal to nil, sort pushdown can still occure
+// - If SortPushdown is false, any sort, or limit pushdown won't be performed.
+// - If SortPushdown is false, Sort and Limit fields cannot be set.
+// - If SortPushdown is true, but Sort is equal to nil, sort pushdown can still occure
 // for capped collection if recordID is not set.
```

Comment:
What does it mean?

Reply:
It means that sort pushdown can be present even if Sort field is not set in one specific case. 
Also, I've fixed the typo.